src/core/math/quat.js

```diff
@@ -114,6 +114,24 @@ class Quat {
         return ((this.x === rhs.x) && (this.y === rhs.y) && (this.z === rhs.z) && (this.w === rhs.w));
     }
 
+    /**
+     * Reports whether two quaternions are equal using an absolute error tolerance.
+     *
+     * @param {Quat} rhs - The quaternion to be compared against.
+     * @param {number} [epsilon] - The maximum difference between each component of the two quaternions.
```

Comment:
Please document the default value for epsilon.

Reply:
Done